### DISCUSSION TITLE
Sending status to multiple contacts

### DISCUSSION THREAD

**Discussion by TheOneironaut** [devlikeapro/waha#582] [Q&A]
I have 60K contacts. Can sending be smooth through this library? I tried some other libraries and it didn't work well there

**Comment by devlikepro** [comment]
Hi! yeah, it's possible only in batches right now
So you need to iterate over 60K contacts and send like 3K contacts at once... It'll create 20 stories on your phone tho.
The issue we'll work on soon :crossed_fingers: https://github.com/devlikeapro/waha/issues/413

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by devlikepro** [reply]
comment with +1 there to get notification when it's ready!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by TheOneironaut** [reply]
Thanks for the answer.
Now I noticed that the bug is marked "NOWEB".
Is this possible in the "WEBJS" engine?

**Reply by devlikepro** [reply]
Nope :( 
https://waha.devlike.pro/docs/how-to/engines/#status
![image](https://github.com/user-attachments/assets/cc0598dc-4dc1-4f40-a839-5661eabdc39c)

Feel free to create a feature request, so we can see how many people are interested in that feature in WEBJS!
https://github.com/devlikeapro/waha/issues/new/choose


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
